common/api-review/auth-exp.api.md

```diff
@@ -250,6 +250,9 @@ export const browserSessionPersistence: Persistence;
 // @public
 export function checkActionCode(auth: Auth, oobCode: string): Promise<ActionCodeInfo>;
 
+// @public
+export function clearCustomTokenProvider(auth: Auth): void;

```

Comment:
Should this bump the changeset? Not sure what the process is for doing so

Reply:
No, though we will want to assign @egilmorez to review the doc comments